Test Runner/UI Tests/FeatureFormViewTests.swift

```diff
@@ -1845,6 +1845,521 @@ final class FeatureFormViewTests: XCTestCase {
+            "The \"Low Voltage\" candidate doesn't exist."
+        )
+        
+        lowVoltageButton.firstMatch.tap()
+        
+        XCTAssertTrue(
+            newAssociationText.waitForExistence(timeout: 5),
+            "The New Association header text doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            associationTypeLabel.waitForExistence(timeout: 5),
+            "The association type label doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            connectivityLabel.waitForExistence(timeout: 5),
+            "The association type value doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            fromElementLabel.waitForExistence(timeout: 5),
+            "The from element label doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            electricDistributionDeviceLabel.waitForExistence(timeout: 5),
+            "The from element value doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            terminalLabel.waitForExistence(timeout: 5),
+            "The terminal label doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            terminalPicker.waitForExistence(timeout: 5),
+            "The terminal picker doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            toElementLabel.waitForExistence(timeout: 5),
+            "The to element label doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            lowVoltageLabel.waitForExistence(timeout: 5),
+            "The to element value doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            fractionAlongEdgeLabel.waitForExistence(timeout: 5),
+            "The fraction along edge label doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            fractionAlongEdgeValueBefore.waitForExistence(timeout: 5),
+            "The fraction along edge label doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            fractionAlongEdgeSlider.waitForExistence(timeout: 5),
+            "The fraction along edge slider doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            addButton.waitForExistence(timeout: 5),
+            "The add button doesn't exist."
+        )
+        
+        terminalPicker.tap()
+        
+        XCTAssertTrue(
+            terminalHighButton.waitForExistence(timeout: 5),
+            "The High terminal option doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            terminalLowButton.waitForExistence(timeout: 5),
+            "The Low terminal option doesn't exist."
+        )
+        
+#if swift(<6.2) && !targetEnvironment(macCatalyst)
+        XCTExpectFailure("""
+            The terminal selection options can be found, but tapping on them
+            doesn't work correctly in Xcode 16.4 for iOS.
+        """)
+#endif
+        terminalLowButton.tap()
+        
+        fractionAlongEdgeSlider.adjust(toNormalizedSliderPosition: 0.75)
+        
+        XCTAssertTrue(
```

Comment:
The test throws an error here when I run it on iOS 26, while passes on iOS 18. No need to fix for now, but may be worth taking a look and see if it can pass on 26 as well.